README.md

```diff
@@ -77,5 +77,19 @@ docker run --env-file .env todo_app
 test container run using CI pipeline in My-CI-Pipeline.yml file everytime you open or reopen a pull request or push to repository
 can also be observed on GitHub workflows to see which ones pass and fail.
 
+## Terraform
+main.tf file uses declarative script to make Azure Infrastructure, where Todoapp is deployed to.
+new variables added 
+ARM_CLIENT_ID
+ARM_CLIENT_SECRET
+ARM_SUBSCRIPTION_ID
+ARM_TENANT_ID
+application can be seen on https://dani-m12-todoapp.azurewebsites.net
+
+## lOGGING
+loggly is used to add logs to the todoapp
+env variable Called lOG_LEVEL, configurably set to DEBUG or ERROR 
```

Comment:
It's surprising to see only `DEBUG and ERROR` listed as options, as LOG_LEVEL can have more values, e.g. WARN, INFO. If there is a specific reason why only these two are useful, please add it to the documentation. Note that some libraries you may import have logging implemented as well and that may be at different level than your listed two.

nitpitck: check the capitalization